packages/web/components/TechnologyForm/MapAndAttachments/ImagesPreview.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { FiTrash2, FiImage } from 'react-icons/fi';
+
+import { IconRow, Media, Button, RadioWrapper } from './styles';
+
+const ImagesPreview = ({ previewedImgFiles, value, onChange, deleteAttachment }) => {
+	return (
+		previewedImgFiles?.map((element, index) => (
+			<IconRow key={element.url} alignItems="flex-start">
+				<RadioWrapper>
+					<input
+						id={element.url}
+						type="radio"
+						name="thumbnail_id"
+						value={element.id}
+						defaultChecked={value === element.id || !index}
+						onChange={onChange}
+					/>
+					{/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+					<label htmlFor={element.url}>
+						<FiImage fontSize="1.6rem" />
+						Usar como capa
+					</label>
```

Comment:
```suggestion
					<label htmlFor={element.url}>
					<input
						id={element.url}
						type="radio"
						name="thumbnail_id"
						value={element.id}
						defaultChecked={value === element.id || !index}
						onChange={onChange}
					/>
						<FiImage fontSize="1.6rem" />
						Usar como capa
					</label>
```
In order to avoid the eslint message, you can wrap the input with the label tag 💡 .